api/nodes/views.py

```diff
@@ -25,7 +26,11 @@ def get_node(self):
         obj = get_object_or_404(Node, self.kwargs[self.node_lookup_url_kwarg])
         # May raise a permission denied
         self.check_object_permissions(self.request, obj)
-        return obj
+
+        if obj.is_deleted:
```

Comment:
Yeah, but with our tendency away from consistency, my concern was more with generating a series of if statements. Practically, though, we only have a couple of models right now, and if we can encourage any future models to use is_deleted for their "no longer active" state, then it's not a big deal. For this particular case, I'm good with handling it all in here as Steve suggests.